plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestReadRowsHelper.java

```diff
@@ -64,33 +59,26 @@ public void testRetryOfSingleFailure()
         batch2.addResponse(ReadRowsResponse.newBuilder().setRowCount(11).build());
 
         List<ReadRowsResponse> responses = ImmutableList.copyOf(
-                new MockReadRowsHelper(client, request, 3, ImmutableList.of(batch1, batch2))
+                new MockReadRowsHelper(client, "test", 3, ImmutableList.of(batch1, batch2))
                         .readRows());
 
         assertThat(responses.size()).isEqualTo(2);
         assertThat(responses.stream().mapToLong(ReadRowsResponse::getRowCount).sum()).isEqualTo(21);
     }
 
-    private static ReadRowsRequest.Builder newRequest()
-    {
-        return ReadRowsRequest.newBuilder().setReadPosition(
-                StreamPosition.newBuilder().setStream(
-                        Stream.newBuilder().setName("test")));
-    }
-
     private static final class MockReadRowsHelper
             extends ReadRowsHelper
     {
         Iterator<MockResponsesBatch> responses;
 
-        MockReadRowsHelper(BigQueryStorageClient client, ReadRowsRequest.Builder request, int maxReadRowsRetries, Iterable<MockResponsesBatch> responses)
+        MockReadRowsHelper(BigQueryReadClient client, String stream, int maxReadRowsRetries, Iterable<MockResponsesBatch> responses)
         {
-            super(client, request, maxReadRowsRetries);
+            super(client, stream, maxReadRowsRetries);
             this.responses = responses.iterator();
         }
 
         @Override
-        protected Iterator<ReadRowsResponse> fetchResponses(ReadRowsRequest.Builder readRowsRequest)
+        protected Iterator<ReadRowsResponse> fetchResponses(long offset)
```

Comment:
can you add a comment indicating why the offset is ignored?

Reply:
@kmjung bump

Reply:
I'm not sure what to say here, really? It's ignored because the class is a mock and needs to match the interface of the parent.

Reply:
Oops, I didn't notice where the class was used. Thanks. Ignore the comment.